migrations/20160912132241_create_risks_of_bias.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+exports.up = (knex) => {
+  const schema = knex.schema;
+
+  schema.createTable('risk_of_biases', (table) => {
+    table.uuid('id').primary();
+
+    table.uuid('trial_id')
+      .notNullable()
+      .references('trials.id');
+    table.text('source_id')
+      .notNullable()
+      .references('sources.id');
+    table.text('source_url')
+      .notNullable();
```

Comment:
The reasoning behind `source_url` and `study_id` is that we should be able to identify an existing risk of bias uniquely. In order to identify it we have to know which review file it came from and to what study it was attributed. So, `source_url` is a reference to the review file and `study_id` something that identifies the risk of bias attributed to a certain study among other review results present in that review file.